build/emtests.rs

```diff
@@ -98,10 +137,26 @@ pub fn build() {
 
     let mut modules: Vec<String> = Vec::new();
     // modules.reserve_exact(TESTS.len());
-    for test in TESTS.iter() {
-        let moudle_name = compile(test);
-        modules.push(format!("mod {};", moudle_name));
+
+    let ignores = read_ignore_list();
+
+    for ext in EXTENSIONS.iter() {
+        for entry in glob(&format!("emtests/*.{}", ext)).unwrap() {
```

Comment:
Is it possible to remove the double loop to do something like this instead?

```rust
        for entry in glob("emtests/*.(c|cpp)").unwrap() {
```

Reply:
I've tried that and `{c,cpp}`. I don't believe this is supported yet for the glob crate:
https://github.com/rust-lang-nursery/glob/blob/master/tests/glob-std.rs

Reply:
This one perhaps?
```
        for entry in glob("emtests/*.cp?p?)").unwrap() {
```

However, it opens the possibility of getting unwanted `abc.cp` files

Reply:
And not that clean. maybe we can keep the double loop for now.